apps/server/cmd/server/main.go

```
@@ -192,3 +228,7 @@ func serve(ctx context.Context, server *http.Server, listener net.Listener) erro
 		return server.Shutdown(shutdownCtx)
 	}
 }
+
+func normalizeAddress(address string) string {
```

golangci-lint (2.13.2)

[error] func normalizeAddress is unused

(unused)